mix.exs

```diff
@@ -27,18 +29,23 @@ defmodule BlogitWeb.Mixfile do
   defp elixirc_paths(_),     do: ["lib"]
 
   defp deps do
-    [{:phoenix, "~> 1.3"},
+    [{:phoenix, "~> 1.4"},
      {:phoenix_pubsub, "~> 1.0"},
      {:phoenix_html, "~> 2.6"},
      {:phoenix_live_reload, "~> 1.0", only: :dev},
      {:gettext, "~> 0.11"},
-     {:cowboy, "~> 1.0"},
+     {:poison, "~> 3.1.0"},
```

Comment:
Why do you need poison if Jason is used now?

Reply:
Poison been reference in endpoint.ex at 2018, if not adding this will not compile. I will remove the Poison at endpoint.ex and replace with Jason.